### PR TITLE
Implementation of this week tasks on the dialog

### PR DESCRIPTION
**Pull request by deaspo**
Almost completing the dialog, with enable/disable okay, ToScript. Help also now points to the right direction. Also reset works

### CODE REVIEW

File: instat/static/InstatObject/R/instat_object.R [hunk 1]
```diff
@@ -373,8 +373,6 @@ instat_obj$methods(delete_dataframe = function(data_name) {
   if(!data_name %in% names(data_objects)) stop(paste("dataframe: ", data_name, " not found"))
   
   data_objects[[data_name]]<<-NULL
-  .self$get_metadata_changed(TRUE)
-  .self$get_data_changed(TRUE)
```
Comment (dannyparsons) [inline instat/static/InstatObject/R/instat_object.R]: You are right to remove these lines but they should be replaced with something to indicate the object has been changed.
@volloholic We only have a data_changed boolean for data objects, so how do we indicate the object has been changed when a data_object is deleted? Do we need a data_changed boolean for the Instat_object too?

Comment (dannyparsons) [inline instat/static/InstatObject/R/instat_object.R]: Steve has fixed this method now. If you remove this change from your pull request I can accept the rest. Thanks.

Comment (deaspo) [inline instat/static/InstatObject/R/instat_object.R]: Ok, let me do that.

**Comment by deaspo** [comment]
Have fixed it

**Pull request merged**